### PR TITLE
Impliment hover

### PR DESCRIPTION
**Pull request by tarunnjoshi**

### CODE REVIEW

File: src/Components/Navbar/navbar.css [hunk 1]
```diff
@@ -21,6 +21,46 @@
     margin-right: 40px;
 }
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:27]: remove space

File: src/Components/Navbar/navbar.css [hunk 2]
```diff
@@ -21,6 +21,46 @@
...
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #031D44;
+    padding: 20px 0px;
+    margin: 0px 20px;
+    display: inline-block;
+    position: relative;
+    opacity: 0.75;
+  }
+
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css]: remove space

File: src/Components/Navbar/navbar.css [hunk 3]
```diff
@@ -21,6 +21,46 @@
...
+  }
+  
+  .nav-link::before {
+    transition: 300ms;
+    height: 4px;
+    content: "";
+    position: absolute;
+    background-color: #121010;
+  }
+
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:44]: here also

File: src/Components/Navbar/navbar.css [hunk 4]
```diff
@@ -21,6 +21,46 @@
...
+    content: "";
+    position: absolute;
+    background-color: #121010;
+  }
+  
+  .nav-link-ltr::before {
+    width: 0%;
+    bottom: 10px;
+  }
+
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:56]: here also

File: src/Components/Navbar/navbar.css [hunk 5]
```diff
@@ -21,6 +21,46 @@
...
+  .nav-link-ltr::before {
+    width: 0%;
+    bottom: 10px;
+  }
+  
+  .nav-link-ltr:hover::before {
+    width: 100%;
+  }
+
+
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:65]: remove space on 61 and 62

**Review by rohit123-rawat — APPROVED** [review]

File: src/Components/Navbar/navbar.css [hunk 6]
```diff
@@ -2,25 +2,68 @@
...
     margin-left: 40px;
     margin-right: 40px;
 }
 
+.navbar {
+    border-bottom: 1px solid #b1a8a8;
+}
+
+.nav-link {
+font-weight: bold;
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:33]: indentation

File: src/Components/Navbar/navbar.css (hunk 2 above)
Comment (tarunnjoshi) [reply src/Components/Navbar/navbar.css]: But it look's good,  have a look on it

File: src/Components/Navbar/navbar.css [hunk 7]
```diff
@@ -2,25 +2,68 @@
...
+color: #031D44;
+padding: 20px 0px;
+margin: 0px 20px;
+display: inline-block;
+position: relative;
+opacity: 0.75;
+}
+  
+.nav-link:hover {
+opacity: 1;
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:46]: indentation

File: src/Components/Navbar/navbar.css [hunk 8]
```diff
@@ -2,25 +2,68 @@
...
+}
+
+.nav-link::before {
+transition: 300ms;
+height: 4px;
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:51]: indentation

File: src/Components/Navbar/navbar.css [hunk 9]
```diff
@@ -2,25 +2,68 @@
...
+content: "";
+position: absolute;
+background-color: #121010;
+}
+  
+.nav-link-ltr::before {
+width: 0%;
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:58]: indentation

File: src/Components/Navbar/navbar.css [hunk 10]
```diff
@@ -2,25 +2,68 @@
...
+bottom: 10px;
+}
+
+.nav-link-ltr:hover::before {
+width: 100%;
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:63]: indentation

File: src/Components/Navbar/navbar.js [hunk 11]
```diff
@@ -49,8 +49,8 @@ function NavBar() {
           <Navbar.Toggle aria-controls="navbarScroll" />
           <Navbar.Collapse id="navbarScroll">
             <Nav className="navbarScroll" >
-              <Nav.Link href="#" className="home">Home</Nav.Link>
-              <Nav.Link href="#">Product</Nav.Link>
+              <Nav.Link href="#" className="home" class="nav-link nav-link-ltr">Home</Nav.Link>
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.js:52]: ```className="home" class="nav-link nav-link-ltr" ``` use single class

**Pull request merged**